### PR TITLE
Add some OSS projects reference in README file

### PR DESCRIPTION
**Pull request by lontivero**
This PR is for adding some open-sourced projects that use (or are based on) NBitcoin. If well there are probably hundreds of projects referencing NBitcoin in Github, these are by sure the best known. I didn't include projects that I think are using NBitcoin but that I cannot find the source code as SmartBit (is it open source?)

I didn't included neither projects with few forks   or few starts like my BlockchainParser, HBitcoin and others.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,41 @@ NBitcoin is inspired by Bitcoin Core code but provides a simpler object oriented
...
+
+* [Wasabi](http://github.com/zkSNACKs/WalletWasabi): Privacy focused, ZeroLink compliant Bitcoin wallet.
+
+* [StratisBitcoinFullNode](http://github.com/stratisproject/StratisBitcoinFullNode): Bitcoin full node in C# https://stratisplatform.com
+
+* [Breeze](http://github.com/stratisproject/Breeze): Breeze Wallet, the first full-block SPV bitcoin wallet 
+
+* [BlockExplorer](http://github.com/stratisproject/BlockExplorer): A set of projects that can index and query stratis blockchains on the fullnode. 
+
+* [BTCPay](http://github.com/btcpayserver/btcpayserver): A cross platform, self-hosted server compatible with Bitpay API
```
Comment (NicolasDorier) [inline README.md]: BTCPay Server

File: README.md [hunk 2]
```diff
@@ -107,6 +107,41 @@ NBitcoin is inspired by Bitcoin Core code but provides a simpler object oriented
...
+
+* [Metaco-Trader](http://github.com/MetacoSA/Metaco-Trader): Bitcoin Wallet for advanced user based on a NBitcoin.Server 
+
+* [Swarmops](http://github.com/Swarmops/Swarmops): Admin backend for any bitcoin-native or swarm organization http://sandbox.swarmops.com/
+
+* [Nako](http://github.com/CoinVault/Nako): A Bitcoin and Altcoin server api that indexes blockchain transactions and addresses 
+
+* [NBXplorer](http://github.com/dgarage/NBXplorer): NBitcoin Explorer 
+
+* [NElements](http://github.com/NicolasDorier/NElements): Adapting NBitcoin for NElements
```
Comment (NicolasDorier) [inline README.md]: Remove this one, outdated fork. I will add element support in NBitcoin.Altcoins soon.

File: README.md [hunk 3]
```diff
@@ -107,6 +107,41 @@ NBitcoin is inspired by Bitcoin Core code but provides a simpler object oriented
...
+
+* [Zen-Wallet](http://github.com/zenprotocol/zen-wallet): Node and GUI for the Zen Protocol. https://www.zenprotocol.com
+
+* [Metaco-Trader](http://github.com/MetacoSA/Metaco-Trader): Bitcoin Wallet for advanced user based on a NBitcoin.Server 
+
+* [Swarmops](http://github.com/Swarmops/Swarmops): Admin backend for any bitcoin-native or swarm organization http://sandbox.swarmops.com/
+
+* [Nako](http://github.com/CoinVault/Nako): A Bitcoin and Altcoin server api that indexes blockchain transactions and addresses 
+
+* [NBXplorer](http://github.com/dgarage/NBXplorer): NBitcoin Explorer
```
Comment (NicolasDorier) [inline README.md]: `A minimalist UTXO tracker for HD Wallets with bitcoin based altcoin support`

**Comment by knocte** [comment]
Can you add geewallet? http://github.com/diginex/geewallet

**Comment by nopara73** [comment]
Also Lykke is doing a bunch of things with NBitcoin, too: https://github.com/LykkeCity/

**Pull request merged**